src/Sylius/Component/Core/Provider/LastYearSalesSummaryProviderInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Provider;
+
+use Sylius\Component\Core\Dashboard\SalesSummaryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface LastYearSalesSummaryProviderInterface
+{
+    public function getSalesSummary(
+        ChannelInterface $channel,
+        \DateTimeInterface $startDate,
+        \DateTimeInterface $endDate
+    ): array;
```

Comment:
No contract for the output - we should introduce a DTO for this (or use `@psalm-return array<int, int> Years to sales amount`?). DTO preferred though.